sdk/src/main/scala/io/horizen/account/AccountSidechainApp.scala

```diff
@@ -91,22 +90,22 @@ class AccountSidechainApp @Inject()
   val consensusStore = new File(dataDirAbsolutePath + "/consensusData")
 
   // Init all storages
+  protected val sidechainHistoryStorage = new AccountHistoryStorage(
+    registerClosableResource(new VersionedLevelDbStorageAdapter(historyStore, maxConsensusSlotsInEpoch * 2 + 1)),
```

Comment:
The number of version to keep is now  a variable parameter of the VersionedLevelDbStorageAdapter constructor:
https://github.com/HorizenOfficial/Sidechains-SDK/blob/b28741b0f6ef0c320fa511f1b21875a414d6af07/sdk/src/main/scala/io/horizen/storage/leveldb/VersionedLevelDbStorageAdapter.scala#L21
Future SDK configurations may use larger slot number per epoch ​​than the current ones and target a smaller slot coefficient, in this case the number of versions to keep would be too big. In this case the expression used to calculate this value (`maxConsensusSlotsInEpoch * 2 + 1`) need to be updated here and in other places.